testsuite/MDAnalysisTests/analysis/test_rms.py

```diff
@@ -149,23 +150,26 @@ def tearDown(self):
         del self.tempdir
 
     def test_rmsd(self):
-        RMSD = MDAnalysis.analysis.rms.RMSD(self.universe, select='name CA')
-        RMSD.run(step=49)
+        RMSD = MDAnalysis.analysis.rms.RMSD(self.universe, select='name CA',
+                                            step=49)
+        RMSD.run()
         assert_array_almost_equal(RMSD.rmsd, self.correct_values, 4,
                                   err_msg="error: rmsd profile should match" +
                                   "test values")
 
     def test_rmsd_single_frame(self):
-        RMSD = MDAnalysis.analysis.rms.RMSD(self.universe, select='name CA')
-        RMSD.run(start=5, stop=6)
-        single_frame = [[ 5, 5, 0.91544906]]
+        RMSD = MDAnalysis.analysis.rms.RMSD(self.universe, select='name CA',
+                                            start=5, stop=6)
+        RMSD.run()
```

Comment:
You can directly run the rmsd analysis when you create them. The run method returns the object itself.